src/app/containers/InlineLink/index.new.test.jsx

```diff
@@ -13,14 +16,14 @@ const fragmentBlock = (text, attributes = []) => ({
 
 describe('InlineLinkContainer', () => {
   describe('with link matching routes for SPA', () => {
-    shouldShallowMatchSnapshot(
+    shouldMatchSnapshot(
       'should render correctly',
       /*
         for the value it would bring, it is much simpler to wrap a react-router Link in a Router, rather than mock a Router or pass come mocked context.
       */
       <StaticRouter>
         <InlineLinkContainer
-          locator="/news/articles/c0000000027o"
+          locator="https://www.bbc.com/news/articles/c0000000027o"
```

Comment:
Following the previous comment could we also add a test for `https://www.test.bbc.com/news/articles/c0000000027o`